sidebars.js

```diff
@@ -72,5 +72,6 @@ module.exports = {
     },
     "warnings",
     "errors",
+    "showcase",
```

Comment:
This should not go under the "Documentation" sidebar, but rather to the top here:

![image](https://user-images.githubusercontent.com/18369201/152340551-10c988e6-1555-43f7-9633-6bacbed38d1b.png)